src/Core/src/Platform/iOS/MauiUIApplicationDelegate.cs

```diff
@@ -51,6 +44,22 @@ public override bool FinishedLaunching(UIApplication application, NSDictionary l
 			return true;
 		}
 
+		UIWindow CreateNativeWindow()
```

Comment:
should we have a virtual hook here as well so users can return a custom UIWindow?

Reply:
Yeah, I will do that in a separate PR when I get android to request windows properly. I want to get a nice way with the bundles/intents.